lib/widgets/user_profile.dart

```diff
@@ -140,49 +140,42 @@ class _UserOverview extends HookWidget {
         textColorBasedOnBackground(theme.colorScheme.secondary);
 
     return Stack(
+      fit: StackFit.expand,
       children: [
         if (userView.person.banner != null)
-          // TODO: for some reason doesnt react to presses
           FullscreenableImage(
             url: userView.person.banner!,
             child: CachedNetworkImage(
+              fit: BoxFit.cover,
               imageUrl: userView.person.banner!,
               errorBuilder: (_, ___) => const SizedBox.shrink(),
             ),
           )
```

Comment:
since you're editing user profile could you also change it so that the background image fits better into the space that it has?